src/bilder/components/caddy/steps.py

```diff
@@ -87,7 +87,7 @@ def install_caddy(caddy_config: CaddyConfig, state=None, host=None):
     files.put(
         name="Configure systemd to load environment variables from file",
         dest="/etc/systemd/system/caddy.service",
-        src=Path(__file__).parent.joinpath("templates", "caddy.service.override"),
+        src=Path(__file__).parent.joinpath("templates", "caddy.service.j2"),
```

Comment:
The override file is intended to be loaded here, the problem is that it needs to go under `/etc/systemd/system/caddy.service.d/env_override.conf`